kehittamo-share-buttons.php

```diff
@@ -30,69 +30,86 @@
-    define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_FB_APP_TOKEN_TRANSIENT', 'kehittamo_share_buttons_fb_app_token' );
-    define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_FB_APP_DEFAULT_API_VERSION', 'v2.7' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_SLUG', 'kehittamo-share-buttons' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_SETTINGS_NAME', 'kehittamo_share_buttons_settings' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_SETTINGS_PAGE_NAME', 'kehittamo-share-buttons-admin' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_VISIBLE_POST_TOP', 'share_buttons_visible_post_top' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_VISIBLE_POST_BOTTOM', 'share_buttons_visible_post_bottom' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_POST_META_KEY', '_kehittamo_share_buttons_post_share_count' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_TRANSIENT_PREFIX_KEY', 'kehittamo_total_shares_count_' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_FB_APP_ID', 'kehittamo_share_buttons_fb_app_id' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_FB_APP_SECRET', 'kehittamo_share_buttons_fb_app_secret' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_FB_APP_TOKEN_TRANSIENT', 'kehittamo_share_buttons_fb_app_token' );
+	define( 'Kehittamo\Plugins\ShareButtons\SHARE_BUTTONS_FB_APP_DEFAULT_API_VERSION', 'v2.7' );
 
 class Load {
 
-    /**
-     * Construct the plugin
-     */
-    function __construct() {
+	/**
+	 * Construct the plugin
+	 */
+	function __construct() {
 
-        add_action( 'plugins_loaded', array( $this, 'load_plugin' ) );
+		add_action( 'plugins_loaded', array( $this, 'load_plugin' ) );
 
-        add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
 
-        add_action( 'wp_enqueue_scripts', array( $this, 'wp_enqueue_scripts' ) );
+		add_action( 'wp_enqueue_scripts', array( $this, 'wp_enqueue_scripts' ) );
 
-        register_activation_hook( __FILE__, array( $this, 'init_plugin' ) );
+		register_activation_hook( __FILE__, array( $this, 'init_plugin' ) );
 
-        register_deactivation_hook( __FILE__, array( $this, 'deactivate_plugin' ) );
+		register_deactivation_hook( __FILE__, array( $this, 'deactivate_plugin' ) );
 
-    }
+	}
 
-    /**
-     * Enable plugin by default
-     */
-    function init_plugin() {
-        $options = get_option( SHARE_BUTTONS_SETTINGS_NAME );
-        if ( ! $options ) {
-            $default_settings = array(
-                SHARE_BUTTONS_VISIBLE_POST_TOP    => 1,
-                SHARE_BUTTONS_VISIBLE_POST_BOTTOM => 1,
-            );
-            update_option( SHARE_BUTTONS_SETTINGS_NAME, $default_settings );
-        }
-    }
+	/**
+	 * Enable plugin by default
+	 */
+	function init_plugin( $network_wide ) {
+
+		function set_defaults() {
```

Comment:
@mkaski function inside a function? So please move`set_defaults` outside the `init_plugin` function and update the PR, thanks! :)